src/main/plugins/everything-plugin/everything-searcher.ts

```diff
@@ -14,14 +14,14 @@ export function everythingSearcher(
     return new Promise((resolve, reject) => {
         const searchTerm = userInput.replace(everythingSearchOptions.prefix, "").trim();
         const utf8Encoding = "cmd /c chcp 65001>nul &&";
-        const command = `${utf8Encoding} "${everythingSearchOptions.pathToEs}" -max-results ${everythingSearchOptions.maxSearchResults} ${searchTerm}`;
+        const command = `${utf8Encoding} "${everythingSearchOptions.pathToEs}" -max-results ${everythingSearchOptions.maxSearchResults} ${getEverythingSearchTerm(searchTerm)}`;
```

Comment:
Would it also work if we just put the search term into double or single quotes?

For example like this:
```typescript
const command = `${utf8Encoding} "${everythingSearchOptions.pathToEs}" -max-results ${everythingSearchOptions.maxSearchResults} "${searchTerm}"`;
```

Reply:
This would disable the everything syntax and treat the query passed to es command line as one string.

After further reading of everything docs, if a users uses the es command line interface in the terminal, they have to escape these special characters manually. 

Now I am not sure if this PR is valid anymore, I mean without this PR, users can still escape these special characters like this.
![image](https://user-images.githubusercontent.com/48618675/91706565-d9297180-eb7e-11ea-8456-d1996d14f342.png)

While this works fine, it might be tedious a bit for users of the regular everything, they are not used to escape these characters.

Reply:
There is also escaping dashes and spaces using double quotes,

> Use double quotes to escape spaces and switches.

but if ueli escaped the spaces and dashes automatically there is no way for users to pass switches. 

So on one hand, ueli can escape these characters and the plugin would be just like the app, on the other hand, users have to manually escape these characters if they want, which would make the plugin just like the command line interface.